resource/decker/support/plugins/a11y/a11y.js

```diff
@@ -65,9 +70,33 @@ const Plugin = {
   id: "a11y",
   init: (reveal) => {
     Reveal = reveal;
-    fixTabsByInert();
+    // This may no longer be necessary if we clearly recommend the handout mode to people using assistive technology
+    // fixTabsByInert();
     addFlyingFocusCallbacks();
     addCustomSpacebarHandler();
```

Comment:
if this mode does not automatically trigger the handout mode, then it should also disable all autoplay videos (and add controls)